iscsi-initiator-utils not to use static

Upstream-status: Backport
This patch is from fedora17.

Signed-off-by: Yao Zhao <yao.zhao@windriver.com>
Signed-off-by: Vu Tran <vu.tran@windriver.com>

diff --git a/usr/Makefile b/usr/Makefile
index 673b7f1..fd14a10 100644
--- a/usr/Makefile
+++ b/usr/Makefile
@@ -61,7 +61,7 @@ iscsiadm: $(ISCSI_LIB_SRCS) $(DISCOVERY_SRCS) iscsiadm.o session_mgmt.o
 
 iscsistart: $(ISCSI_LIB_SRCS) $(INITIATOR_SRCS) $(FW_BOOT_SRCS) \
 		iscsistart.o statics.o
-	$(CC) $(CFLAGS) -static $^ -o $@
+	$(CC) $(CFLAGS) $^ -o $@
 clean:
 	rm -f *.o $(PROGRAMS) .depend $(LIBSYS)
 
